java/examples/build.gradle

```
@@ -6,20 +6,20 @@ plugins {
 repositories {
     // Use Maven Central for resolving dependencies.
     mavenCentral()
+    mavenLocal()
 }
 
 dependencies {
-    implementation project(':client')
+    implementation 'software.amazon:glide:0.0.1'
 
     implementation 'redis.clients:jedis:4.4.3'
     implementation 'io.lettuce:lettuce-core:6.2.6.RELEASE'
     implementation 'commons-cli:commons-cli:1.5.0'
 }
 
-run.dependsOn ':client:buildRustRelease'
+run.dependsOn ':client:publishToMavenLocal'
```

I think this should fix the build
```suggestion
compileJava.dependsOn ':client:publishToMavenLocal'
```

---

I already fixed it. I accidentally deleted the groupId and artifactId from the build.gradle.